corehq/apps/cloudcare/touchforms_api.py

```diff
@@ -74,14 +74,8 @@ def filter_cases(self, xpath, additional_filters=None, auth=None, extra_instance
             "session_data": session_data,
         }
 
-        if use_formplayer:
-            url = settings.FORMPLAYER_URL + "/filter_cases"
```

Comment:
based on my read it looks like the new code in `post_data` would use `touchcare-filter-cases` from the `"action"` above instead of this URL. given that it's still an R&D feature and cloudcare is broken for our biggest project on prod I wasn't too worried about breaking that.